packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1305,6 +1307,18 @@ export class TransactionController extends BaseControllerV1<
     this.trackingMap.delete(networkClientId);
   }
 
+  #stopAllTracking() {
+    this.pendingTransactionTracker.stop();
+    this.removePendingTransactionTrackerListeners();
+    this.incomingTransactionHelper.stop();
+    this.removeIncomingTransactionHelperListeners();
+    this.pendingTransactionTracker.stop();
+
+    for (const [networkClientId] of this.trackingMap) {
```

Comment:
```suggestion
    for (const [networkClientId] of Object.entries(this.trackingMap)) {
```

Reply:
so I think this is correct as is?

Reply:
I think this is right as is:

```
const trackingMap = new Map()
trackingMap.set('key1', 'foo')
trackingMap.set('key2', 'bar')
for (const [key, value] of trackingMap) {
  console.log(key, value)
}
```
yields
```
key1 foo
key2 bar
```